hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/main/java/org/apache/hadoop/yarn/server/federation/store/FederationStateStore.java

```diff
@@ -60,7 +60,21 @@ public interface FederationStateStore extends
    * Load the version information from the federation state store.
    *
    * @return the {@link Version} of the federation state store
+   * @throws Exception an exception occurred in load version.
    */
-  Version loadVersion();
+  Version loadVersion() throws Exception;
```

Comment:
Thank you for helping to review the code, I will add tests, but this pr is a framework, I will add some simple tests. It will be implemented in `MemoryFederationStateStore`, `SQLFederationStateStore`, and `ZookeeperFederationStateStore` later. When these functions are completed, I will add more test cases.